packages/react-reconciler/src/ReactFiberNewContext.new.js

```diff
@@ -311,6 +354,271 @@ export function propagateContextChange<T>(
   }
 }
 
+function propagateContextChanges<T>(
+  workInProgress: Fiber,
+  contexts: Array<any>,
+  renderLanes: Lanes,
+  forcePropagateEntireTree: boolean,
+): void {
+  // Only used by lazy implemenation
+  if (!enableLazyContextPropagation) {
+    return;
```

Comment:
Should this throw or are you calling this function in the old impl?

Reply:
Since this is probably for GCC stripping it, I'm specifically asking if we should throw here if GCC doesn't strip it for some reason.

Reply:
Throwing isn't dead code so it doesn't help dead code elimination. It only helps indirectly by forcing us to wrap the caller in a condition. But I prefer to wrap in both places.

Reply:
Yeah, for me that's a feature not a bug. If this were to accidentally be included, it would silently fail this way.